ApollosProject/apollos-prototype
# Fixes StretchyView bug on overscroll effect

## DESCRIPTION
<img src="https://user-images.githubusercontent.com/2053547/72188266-e1eaee80-33c7-11ea-8bb7-be00d4ddf24a.gif" width="30%" /><img src="https://user-images.githubusercontent.com/2053547/72189425-c503ea80-33ca-11ea-907a-13c9c0a62823.gif" width="30%" />

### What does this PR do, or why is it needed?
This PR fixes a bug in StretchyView that was causing a crash on iOS. It does this by:
- correcting an api syntax error
- using `Animated.ScrollView`

Coincidentally, this might actually improve `StretchyView` performance on iOS by actually passing its animation to the native thread, where it might not have been doing so previously.

### What design trade-offs/decisions were made?
Additionally, this PR adds a commit (92c0151), **that you are welcome to keep or revert**, that removes all `StretchyView` functionality from Android. It basically just renders a `View` with no additionally renders or computational logic. This is because `StretchyView` doesn't actually work on Android to begin with because Android doesn't overscroll. No overscroll, no way to animate the image with a stretchy overscroll effect. So might as well ditch all the code right 🤷‍♂ Like I said, keep it or revert it. I leave that decision to the reviewer.

### How do I test this PR?
- open app
- tap on an Article/Sermon/Series (anything but a Devo single)
- everything look correct?
- pull down to see the overscroll stretchy effect on ios and scroll on android.
- No error/crash? We're good!

### What automated tests did you write?
All the changes in this PR are unable to be covered with jest snapshots.

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
